core-feature-pack/src/main/resources/content/bin/jboss-cli.xml

```diff
@@ -47,12 +47,24 @@
 
     <!-- Include the prompt with the command into the output for each command executed in non-interactive mode -->
     <echo-command>false</echo-command>
-    
+
     <!-- Uncomment to set the command timeout. Element value is in seconds -->
     <!-- <command-timeout>30</command-timeout> -->
 
     <!-- Uncomment to display operation responses using JSON syntax.
     By default responses are displayed using DMR string syntax. -->
     <!-- <output-json>true</output-json> -->
-    
+
+    <!-- Uncomment to display to configured CLI color output. -->
+    <!-- Available colors: black, red, green, yellow, blue, magenta, cyan, white and default, which is the terminal's default foreground color-->
+    <!--<color-output>-->
```

Comment:
A single comment for the whole element would make un-commenting easier.

Reply:
Something like this?
```xml
    <!-- Uncomment to display to configured CLI color output.
    Available colors: black, red, green, yellow, blue, magenta, cyan, white and default, which is the terminal's default
    foreground color-->
    <!--
    <color-output>
        <enabled>true</enabled>
        <error-color>red</error-color>
        <warn-color>yellow</warn-color>
        <success-color>default</success-color>
        <required-color>magenta</required-color>
        <workflow-color>green</workflow-color>
        <prompt-color>blue</prompt-color>
    </color-output>
    -->
```